docs/pages/experiments/slider-inverted-fill.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import * as Slider from '@base_ui/react/Slider';
+import { Styles, TrackFill } from './slider';
+
+export default function App() {
+  return (
+    <div className="App">
+      <Slider.Root className="MySlider" defaultValue={50}>
+        <Slider.Output className="MySlider-output" />
+        <Slider.Track className="MySlider-track">
+          <TrackFill className="MySlider-track-fill" inverted />
+          <Slider.Thumb className="MySlider-thumb one" />
+        </Slider.Track>
+      </Slider.Root>
+
+      <Slider.Root className="VerticalSlider" defaultValue={50} orientation="vertical">
```

Comment:
In this demo it's pretty hard to change the value by clicking on a track. The click area could be larger than the visible track.

Reply:
I've updated the vertical slider styles in `/experiments`, the latest styles with a wider vertical track area were already in the docs, just didn't copy them back to here 😓